test/test-lib/fake-device.ts

```diff
@@ -96,5 +96,25 @@ export async function provisionDevice(
 		return state;
 	};
 
+	device.patchStateV2 = async (): Promise<null> => {
+		const stateToPatch: DeviceState = {
+			local: {
+				supervisor_version: '5.0.1',
+				name: device.name,
+				config: {},
+				apps: {},
+			},
+			dependent: {
+				apps: {},
+				devices: {},
+			},
+		};
+		await supertest(app, device)
+			.patch(`/device/v2/${device.uuid}/state?apiKey=${device.token}`)
```

Comment:
I don't think the `apiKey` arg is necessary as `supertest(app, device)` should use the auth header already, and I think that's how the supervisor patches @CameronDiver ?
```suggestion
		await supertest(app, device)
			.patch(`/device/v2/${device.uuid}/state`)
```